examples/tracer-web/package.json

```diff
@@ -27,25 +30,29 @@
   "devDependencies": {
     "@babel/core": "^7.6.0",
     "babel-loader": "^8.0.6",
-    "ts-loader": "^6.0.4",
-    "webpack": "^4.35.2",
-    "webpack-cli": "^3.3.9",
-    "webpack-dev-server": "^3.8.1",
-    "webpack-merge": "^4.2.2"
+    "ts-loader": "^9.2.6",
+    "typescript": "^4.5.2",
+    "webpack": "^5.65.0",
+    "webpack-cli": "^4.9.1",
+    "webpack-dev-server": "^4.6.0",
+    "webpack-merge": "^5.8.0"
   },
   "dependencies": {
     "@opentelemetry/api": "^1.0.2",
-    "@opentelemetry/context-zone": "0.25.0",
-    "@opentelemetry/core": "0.25.0",
-    "@opentelemetry/exporter-otlp-http": "0.25.0",
-    "@opentelemetry/exporter-zipkin": "0.25.0",
-    "@opentelemetry/instrumentation": "0.25.0",
-    "@opentelemetry/instrumentation-fetch": "0.25.0",
-    "@opentelemetry/instrumentation-xml-http-request": "0.25.0",
-    "@opentelemetry/sdk-metrics-base": "0.25.0",
-    "@opentelemetry/propagator-b3": "0.25.0",
-    "@opentelemetry/sdk-trace-base": "0.25.0",
-    "@opentelemetry/sdk-trace-web": "0.25.0"
+    "@opentelemetry/context-zone": "^1.0.1",
+    "@opentelemetry/core": "^1.0.1",
+    "@opentelemetry/sdk-trace-base": "^1.0.1",
+    "@opentelemetry/sdk-trace-web": "^1.0.1",
```

Comment:
Why are some dependencies pinned but ohters are not?

Reply:
Nice catch, this is inconsistent. I was going for experimental/non-production -> pin and production don't pin.

But as an example I'm happy to pin everything?

Reply:
I also reordered the imports so that all production imports are now grouped

Reply:
I would pin everything to ensure it keeps working. Or at least move to `~` instead `^`.

Experimental packages implement interface from SDK so if something is added there it may cause issues.